packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx

```diff
@@ -81,8 +81,8 @@ const COLUMNS = [
 ];
 
 const CREATE_CONFIG = {
-  title: 'Create a new relation between Map Overlay and Map Overlay Group',
   actionConfig: {
+    title: 'New relation between map overlay group and map overlay',
```

Comment:
Had a quick look at the table in Postgres. I realise it can also be a group–group relation—not just a group–overlay relation—but it felt a little unwieldy to include all of that detail

Deferred to the original title here, but open to input